README.md

```diff
@@ -17,6 +17,8 @@ p = PageviewsClient(user_agent="<person@organization.org> Selfie, Cat, and Dog a
 p.article_views('en.wikipedia', ['Selfie', 'Cat', 'Dog'])
 p.project_views(['ro.wikipedia', 'de.wikipedia', 'commons.wikimedia'])
 p.top_articles('en.wikipedia', limit=10)
+p.top_articles_by_country('en.wikipedia', limit=10)
```

Comment:
should probably be `top_by_country`, as this returns a list of countries ordered from highest to lowest traffic on this wiki.  Sorry for the confusing naming of the endpoints, the one below should indeed have been `top_articles_per_country` and this one is really hard to name so maybe `top_by_country` is ok.

Reply:
yep -- the dangers of copy-and-pasting! I updated to `top_by_country` to keep it simple and consistent with the API